proto/openconfig/openconfig_interfaces/gen.go

```diff
@@ -15,7 +15,7 @@
 
 // Package openconfig_interfaces contains code generated from the OpenConfig
 // Interfaces protobuf for Hercules.
-package openconfig_interfaces
+package openconfig_openconfig_interfaces
```

Comment:
Trying to understand what leads to this doubling of "openconfig_" for most/all packages, and if anything can be done about this from a readability POV (it looks a bit odd/artificial) ? Apologies if this was discussed previously and I just cannot recall it. Can the tool be taught to suppress the first openconfig_, or does it lead to unwanted implications?

Reply:
This seems to be affecting the go files only, I see the proto files are following the previous naming approach.

Reply:
Yes - this was just an error in the package naming that came to light when I regenerated the files. It's just for Go -- going forward setting the `go_package` explicitly will remove this stuttering.